Assets/Scripts/Scheduler/EventPrototypes.cs

```diff
@@ -0,0 +1,43 @@
+﻿#region License
```

Comment:
For a better organization, I think that is better to move this file to Models/Prototypes.